sdk/monitor/mgmt/src/main/java/com/azure/management/monitor/Action.java

```diff
@@ -14,7 +15,7 @@
 @JsonTypeInfo(
     use = JsonTypeInfo.Id.NAME,
     include = JsonTypeInfo.As.PROPERTY,
-    property = "odata.type",
+    property = "odata\\.type",
```

Comment:
Did you try to run living test for monitor locally after these changes?

Reply:
Already had a test for this.
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/monitor/mgmt/src/test/java/com/azure/management/monitor/TypeSerializationTests.java#L19